packages/strapi-plugin-i18n/services/locales.js

```diff
@@ -40,6 +40,10 @@ const initDefaultLocale = async () => {
   const existingLocalesNb = await strapi.query('locale', 'i18n').count();
   if (existingLocalesNb === 0) {
     await create(DEFAULT_LOCALE);
+  }
+
+  const defaultLocale = await getDefaultLocale();
```

Comment:
when is this needed ?

Reply:
On a second thought, it is not that useful.
It was in case some locales exist but not the default locale in the core_store.